sms_939/models/recurring_contract_group.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    Copyright (C) 2018 Compassion CH (http://www.compassion.ch)
+#    @author: Quentin Gigon <gigon.quentin@gmail.com>
+#
+#    The licence is in the file __manifest__.py
+#
+##############################################################################
+from odoo import api, models, _
+from odoo.exceptions import UserError
+
+
+class ContractGroup(models.Model):
+    _inherit = 'recurring.contract.group'
+
+    @api.multi
+    @api.onchange('payment_mode_id')
+    def onchange_payment_mode_id(self):
```

Comment:
This method already exists here : https://github.com/CompassionCH/compassion-switzerland/blob/10.0/sponsorship_switzerland/models/contract_group.py#L149

You can delete this file.